elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -100,6 +127,11 @@ private IndexManagementStrategy getIndexManagementStrategy(Properties properties
 		return strategy != null ? IndexManagementStrategy.valueOf( strategy ) : IndexManagementStrategy.NONE;
 	}
 
+	private String getIndexManagementWaitTimeout(Properties properties) {
+		String timeout = properties.getProperty( ElasticsearchEnvironment.INDEX_MANAGEMENT_WAIT_TIMEOUT );
+		return timeout != null ? Integer.valueOf( timeout ) + "ms" : DEFAULT_INDEX_MANAGEMENT_WAIT_TIMEOUT;
```

Comment:
I might be too strict with user input validation, but in my experience people always have huge amounts of issues with our properties.
I've considered moving to a "typesafe" configuration format like XML but having seen how much pain it introduces when the schemas have to be updated for new features or better formats (i.e. Infinispan) I'm glad we're using properties.. we just need to make sure we're paranoid with validation.